config/default.yml

```diff
@@ -6,6 +6,7 @@ Flexport/EngineApiBoundary:
   UnprotectedEngines: []
   StronglyProtectedEngines: []
   EngineSpecificOverrides: []
+  AllowCrossEngineFactoryBotFromEngines: []
```

Comment:
nit: this name would feel slightly more idiomatic in the context of the others: `FactoryBotUnprotectedEngines: []`

Reply:
@maxh I agree in principle, but `UnprotectedEngines` and `AllowCrossEngineFactoryBotFromEngines` actually work quite differently:

`UnprotectedEngines` lists engines that it's OK to **reach into**, whereas `AllowCrossEngineFactoryBotFromEngines` lists engines that are allowed to **reach out** to use other engines' factories. My intention with the more long-winded name was to make that explicit. Happy to change the name if you can think of a better one!

Reply:
Ah, makes sense. What do you think of `FactoryBotOutboundAccessAllowedEngines`? The pattern I'm trying to preserved is "NounVerbedEngines" rather than "VerbNounEngines".

Reply:
Sounds good. I renamed `AllowGlobalFactoryBotFromEngines` to `FactoryBotGlobalAccessAllowedEngines` as well.